aws/init

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash -e
+# Copyright (c) 2023 EPAM Systems, Inc.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+usage() {
+  cat << EOF
+AWS parameters:
+  --aws-region               AWS Region (defaults to value from: AWS_REGION)
+  --aws-profile              AWS Profile (defaults to value from: AWS_PROFILE)
+EOF
+}
+
+while [ "$1" != "" ]; do
+  case $1 in
+    --aws-region )      shift
```

Comment:
This is a good idea to ask user to enter was-region and aws profile. Yet, in this case `ask` should not be executed. Not sure if I have seen this